docs/web_scraping_101/anti_scraping_techniques.md

```diff
@@ -1,41 +1,41 @@
 
-For successful large-scale scrapes, it is essential to have a sufficient pool of proxies and to time the workload to maximize the scraping throughput while not burning your proxies.
+Our research determined that, for successful large-scale scraping activities, it is essential to have a sufficient pool of proxies and to time the workload to maximize the scraping throughput while burning proxies.
 
-[Apify Proxy]({{@link proxy.md}}) enables you to do just this using a pool of datacenter and residential proxies.
+[Apify Proxy]({{@link proxy.md}}) enables you to enhance your data throughput and access websites from any geographical location by using an extensive pool of datacenter and residential proxies.
 
 ## [](#ip-rate-limiting) IP rate limiting
 
-When crawling a website, you’ll typically send many more requests from a single IP address than a human user could generate over the same period. Websites can easily monitor how many requests they receive from a single IP address. If the number of requests exceeds a certain limit, websites can block that IP address or make you pass a [CAPTCHA](https://en.wikipedia.org/wiki/CAPTCHA) test.
+When crawling a website, a webscraping bot will typically send many more requests from a single IP address than a human user could generate over the same period. Websites can easily monitor how many requests they receive from a single IP address. If the number of requests exceeds a certain limit, websites can block that IP address or require a [CAPTCHA](https://en.wikipedia.org/wiki/CAPTCHA) test.
 
 ### [](#bypassing-ip-rate-limiting) Bypassing IP rate limiting
 
-There are two ways to work around rate limiting. One option is to limit how many pages on a single site you scrape concurrently, and possibly even introduce delays (after reaching the original limit). The other is to use proxy servers and rotate IP addresses after a certain number of requests.
+We found that two ways are used to work around rate limiting. One method is to limit how many pages on a single site are scraped concurrently, with delays possibly even being intentionally introduced (after reaching the original limit). The other method is to use proxy servers and rotate IP addresses after a certain number of requests.
 
-To lower the concurrency when using the [Apify SDK](https://sdk.apify.com/docs/typedefs/basic-crawler-options#maxconcurrency), just pass the `maxConcurrency` option to your crawler's setup. If you use actors from [Apify Store](https://apify.com/store), you can usually set the maximum concurrency in the actor's input.
+Apify actors are designed to be able to reduce the workload on websites being scraped. To lower the concurrency when using the [Apify SDK](https://sdk.apify.com/docs/typedefs/basic-crawler-options#maxconcurrency), just pass the `maxConcurrency` option to your crawler's setup. If you use actors from [Apify Store](https://apify.com/store), you can usually set the maximum concurrency in the actor's input.
```

Comment:
Maybe just "... designed to reduce the workload..." here?

Reply:
Go for it 👍